twitter/Models/User.h

```diff
@@ -0,0 +1,22 @@
+//
+//  User.h
+//  twitter
+//
+//  Created by Joey R. Hernandez Perez on 6/29/20.
+//  Copyright © 2020 Emerson Malca. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface User : NSObject
+
+@property (nonatomic, strong) NSString *name;
+@property (nonatomic, strong) NSString *screenName;
```

Comment:
This sounds like Twitter talking. In the real world we usually refer to a twitter handle. (again, not a twitter user). 

Main point is, Do Not allow the server to define your model. It's Your model.